### ISSUE TITLE
Blast core_nt database

### ISSUE THREAD

**Issue by JoeCohen** [MushroomObserver/mushroom-observer#2538]
The Run BLAST link should  point to the core_nt database.
See https://mushroomobserver.slack.com/archives/C040TH9FV/p1732117497825869.
(Requested by @AlanRockefeller.)

**Comment by JoeCohen** [comment]
It appears that the correct url should include this param`DATABASE=core_nt`, e.g.:
```ruby
https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome&DATABASE=core_nt&QUERY=MN820553
```

**Comment by JoeCohen** [comment]
Implemented by #2544, deploy-2024-11-21-23-04
